webpack.config.js

```diff
@@ -7,17 +7,20 @@ module.exports = {
     filename: 'bundle.js',
   },
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.js$/,
-        loader: 'babel-loader',
+        use: {
+          loader: 'babel-loader',
+        },
         exclude: /node_modules/,
       },
-    ],
-    preLoaders: [
+      // Should be a pre-loader and probably b4 babel but can't get it to work :-(
```

Comment:
What's the problem here?

Reply:
I think it actually works as a preloader, I just didn't understand that all the eslint had to pass before the compilation. Currently it stops after compiling and won't re-compile before your error is fixed. I prefer this solution as it is convenient to throw in console.log during debugging :grin:

Reply:
Let's keep it as a preloader, I think this PR is good to go after this change

Reply:
Turns out that it was correct, webpack seems to run the last rule first. Everything fails with pre-loader first but when I add `enforce: 'pre'` it works as expected. This fits docs: https://github.com/webpack-contrib/eslint-loader (although it is slightly outdated with old webpack syntax)